gflib/bg.c

```diff
@@ -252,17 +252,18 @@ static void SetBgAffineInternal(u8 bg, s32 srcCenterX, s32 srcCenterY, s16 dispC
 
     switch (sGpuBgConfigs.bgVisibilityAndMode & 0x7)
     {
+
```

Comment:
Why did you move the `default` and `0` cases to the top of the `switch`, in addition to adding an extra newline?